README.md

```diff
@@ -52,8 +56,8 @@ can validate a particular path configuration, where `n` is the number of tiles?
 In one sentence, explain why.
 
 ## A1.2
-
-«replace this with your answer»
+O(n)
+The parser will load the entire path into memory, then check if element i is equal to element n-i, stopping after i = n-i or i = n-i+1, or if element i != n-i thus the memory requirement will be n.
```

Comment:
# 3

Yes, but... a general purpose parser would not know that it could do that, so it would use a pushdown stack and a state machine. But the answer would be the same: the list of tiles would just end up on the stack.